src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -265,4 +268,51 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
         return state;
     }
+
+    private async _processAdditionalAction(error: unknown, shouldSetAsDefaultInstrument: boolean, instrumentId: string, methodId: string): Promise<InternalCheckoutSelectors> {
+        if (!(error instanceof RequestError) || !some(error.body.errors, {code: 'additional_action_required'})) {
+            return Promise.reject(error);
+        }
+
+        const additionalAction: DigitalRiverAdditionalProviderData = error.body.provider_data;
+
+        if (!this._digitalRiverCheckoutData) {
+            throw new InvalidArgumentError('Unable to proceed because payload payment argument is not provided.');
+        }
+
+        return this._getDigitalRiverJs().authenticateSource({
+            sessionId: this._digitalRiverCheckoutData.sessionId,
+            sourceId: additionalAction.source_id,
+            sourceClientSecret: additionalAction.source_client_secret,
+        }).then((data: DigitalRiverAuthenticateSourceResponse) => {
+            if (data.status === AuthenticationSourceStatus.complete || data.status === AuthenticationSourceStatus.authentication_not_required) {
+                if (!this._digitalRiverCheckoutData) {
+                    throw new InvalidArgumentError('Unable to proceed because payload payment argument is not provided.');
+                }
+
+                const paymentPayload = {
+                    methodId,
+                    paymentData: {
+                        formattedPayload: {
+                            bigpay_token: {
+                                token: instrumentId,
+                            },
+                            verification_value: '000',
```

Comment:
Why are you hardcoding the verification value? shouldnt you be using TSV Verification fields?

Reply:
Because this field is required in Bigpay, but Digital River doesn't support TSV verification and fields.